.github/workflows/full-ci.yml

```diff
@@ -185,7 +185,6 @@ jobs:
       - name: Install dependencies
         run: |
           brew install --overwrite go mage
-          brew install python3
```

Comment:
Do test fail on self-hosted or on github runners?

Reply:
on solf-host, I see this patch can't help